packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -262,11 +263,7 @@ class PerformanceMonitoring {
       'checkBrowserResponsiveness'
     )
 
-    /**
-     * TODO: Refactor captureTimings flag here once this PR
-     * is addressed - https://github.com/elastic/apm-agent-rum-js/issues/334
-     */
-    if (checkBrowserResponsiveness && !tr.captureTimings) {
+    if (checkBrowserResponsiveness && tr.options.managed) {
```

Comment:
Our previous logic was doing this for all transaction except for page-load since we never want to miss sending the page-load. But right now if the browser is not responsive, we still dont send page-load. Can we remove this check?